shell/probe/install.sh

```diff
@@ -110,7 +112,7 @@ EOF
 install_darwin() {
     local _running_user="${USER}"
     local _plist_path="/Users/${_running_user}/Library/LaunchAgents/com.preludesecurity.detect.plist"
-    local _app_dir="/Users/${_running_user}/.prelude/bin"
+    local _app_dir=${PRELUDE_DIR:="/Users/${_running_user}/.prelude/bin"}
```

Comment:
Why not use ~/tmp? This would work on Linux and Mac. Or just $mktemp

Reply:
seems risky to install the probe in tmp, it might get autodeleted there

Reply:
On a phone but.. is this for the permanent location of the probe binary or the initial location to download it to?

Reply:
this is the permanent location

Reply:
kk in that case, ignore me!